package com.desafioitau.api.transferencia.exceptions.conta.exception;

public class ContaClientException extends ContaException {
    public ContaClientException(Throwable cause) {
        super(cause);
    }
}
